src/plusplus.coffee

```diff
@@ -33,7 +33,9 @@ module.exports = (robot) ->
     # from beginning of line
     ^
     # the thing being upvoted, which is any number of words and spaces
-    ([\s\w'@.-]*)
+    ([\s\w'@.-:]*)
```

Comment:
Rather than inside of the matching group, how about a `:?` outside of the group so the colon isn't included in the name? (so `demo: and demo` will both match the same thing)